cumulusci/tasks/apex/testrunner.py

```diff
@@ -168,6 +160,21 @@ def _init_options(self, kwargs):
 
         self.options["managed"] = process_bool_arg(self.options.get("managed", False))
 
+        self.options["retry_errors"] = process_list_arg(
+            self.options.get("retry_errors", [])
+        )
+        try:
+            self.options["retry_errors"] = [
+                re.compile(r) for r in self.options["retry_errors"]
+            ]
+        except re.error as e:
+            raise TaskOptionsError(
+                "An invalid regular expression was provided ({})".format(e)
```

Comment:
Does this error show _which_ expression failed to compile?

Reply:
It shows which token failed, but doesn't identify the source expressed.

Reply:
Can we compile these one at a time instead of in a comprehension so we can include the failed expression in our own error message?